static/js/copyPasteEvents.js

```diff
@@ -1,3 +1,5 @@
+var _ = require("underscore");
```

Comment:
Hum, I thought `require('underscore')` would use the code on `src/node_modules/underscore/`, not `src/static/js/underscore.js`...